devices/pine64-pinephone-braveheart/u-boot/default.nix

```diff
@@ -42,10 +26,12 @@ in
     CONFIG_BOOTDELAY=0
   '';
 }).overrideAttrs(old: rec {
-  version = "2020.07";
-  src = fetchurl {
-    url = "ftp://ftp.denx.de/pub/u-boot/u-boot-${version}.tar.bz2";
-    sha256 = "0sjzy262x93aaqd6z24ziaq19xjjjk5f577ivf768vmvwsgbzxf1";
+  version = "495f85a398272e6d8ea8142790158afa1bb29c77";
```

Comment:
A `version` in Nixpkgs is a software version.

When following an in-development repository we [use the commit date of the most recent change](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming).